src/api/digital-exchange/components.js

```diff
@@ -0,0 +1,15 @@
+import { LIST_DE_COMPONENTS_OK } from 'test/mocks/digital-exchange/components';
+import { makeRequest, METHODS } from '@entando/apimanager';
+
+// eslint-disable-next-line
```

Comment:
please do not use blanket disable statements: specify which rule you are disabling